src/common/AriaQueryHandler.ts

```diff
@@ -0,0 +1,133 @@
+/**
+ * Copyright 2020 Google Inc. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { InternalQueryHandler } from './QueryHandler.js';
+import { ExecutionContext } from './ExecutionContext.js';
+import { ElementHandle } from './JSHandle.js';
+import { Protocol } from 'devtools-protocol';
+import { CDPSession } from './Connection.js';
+import { DOMWorld } from './DOMWorld.js';
+
+function backendNodeIdtoHandle(
+  exeCtx: ExecutionContext,
+  backendNodeId: number
+): Promise<ElementHandle> {
+  return exeCtx._adoptBackendNodeId(backendNodeId);
+}
+
+async function queryAXTree(
+  client: CDPSession,
+  element: ElementHandle,
+  accessibleName?: string,
+  role?: string
+): Promise<Protocol.Accessibility.AXNode[]> {
+  // @ts-ignore
+  const { nodes } = await client.send(
```

Comment:
why the ts-ignore? Is that because the protocol doesn't yet know about `Accessibility.queryAXTree` ?

Reply:
yes exactly. Once we roll Chromium r809590 (or later) we can land this pr without the ts-ignores.